tests/postgres_tests/test_array.py

```diff
@@ -363,6 +365,25 @@ class Meta:
         self.assertEqual(len(errors), 1)
         self.assertEqual(errors[0].id, 'postgres.E002')
 
+    def test_nested_field_checks(self):
+        """
+        #25867 - Nested ArrayFields are permitted.
```

Comment:
I prefer to include the ticket reference only for obscure issues where additional context provided by the ticket is helpful.